### PR TITLE
feat: progress-bar

### PR DESCRIPTION
**Pull request by femacedogomes**
In this PR i create a component progress bar.
task:  https://www.notion.so/028386dd8cc04ffeaa06931d912bd0fc?v=b8d42189aed746d8a0a716c781c8d731&p=26f1d304ef9e4e389d4c236441dab860&pm=s

### CODE REVIEW

File: src/components/atoms/ProgressBar/index.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import cx from 'classnames';
+import React from 'react';
+
+export type ProgressBarProps = {
+  totalsteps?: string;
```
Comment (renanpontez) [inline src/components/atoms/ProgressBar/index.tsx]:
```suggestion
  totalSteps?: string;
```

File: src/components/atoms/ProgressBar/index.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  stepsready?: string;
```
Comment (renanpontez) [inline src/components/atoms/ProgressBar/index.tsx]:
```suggestion
  stepsReady?: string;
```

File: src/components/atoms/ProgressBar/index.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  label?: string;
+};
+
+const ProgressBar: React.FC<ProgressBarProps> = ({
+  totalsteps,
+  stepsready,
+  label,
+}) => {
+  const StepsReadyPercente = (Number(stepsready) / Number(totalsteps)) * 100;
```
Comment (renanpontez) [inline src/components/atoms/ProgressBar/index.tsx]:
```suggestion
  const progressPercentage= (Number(stepsready) / Number(totalsteps)) * 100;
```

File: src/components/atoms/ProgressBar/index.tsx [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+  return (
+    <div>
+      <span className="p-2.5 text-sm font-semibold text-slate-50">
+        {label} {stepsready}/{totalsteps}
+      </span>
```
Comment (renanpontez) [inline src/components/atoms/ProgressBar/index.tsx]: Add a new prop showLabel to do something like:

```suggestion
      {showLabel && (
        <span className="p-2.5 text-sm font-semibold text-slate-50">
          {label} {stepsready}/{totalsteps}
        </span>
       )
```

File: src/components/atoms/ProgressBar/index.tsx [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+      <div className=" h-[8px] w-[150px] rounded-md bg-slate-100">
```
Comment (renanpontez) [inline src/components/atoms/ProgressBar/index.tsx]: Let's try to use the `h-1, h-2, h-3` options from tailwind when possible (https://tailwindcss.com/docs/height).
Same for width. 

But in this case, **width should be** `100%`. The parent (the one who call <ProgressBar/>) will be responsible to manage it's size and positioning in the page.

**Review by renanpontez — CHANGES_REQUESTED** [review]
Left some small comments

File: src/components/atoms/ProgressBar/index.tsx [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+  totalSteps,
+  stepsReady,
+  label,
+  showLabel = true,
+}) => {
+  const progressPercentage = (Number(stepsReady) / Number(totalSteps)) * 100;
+
+  return (
+    <div>
+      {showLabel && (
```
Comment (pedrohbl03) [inline src/components/atoms/ProgressBar/index.tsx:21]:
```suggestion
      {label && (
```

File: src/components/atoms/ProgressBar/index.tsx [hunk 7]
```diff
@@ -0,0 +1,45 @@
+import cx from 'classnames';
+import React from 'react';
+
+export type ProgressBarProps = {
+  totalSteps?: string;
+  stepsReady?: string;
+  label?: string;
+  showLabel?: boolean;
```
Comment (pedrohbl03) [inline src/components/atoms/ProgressBar/index.tsx:8]:
```suggestion
```

File: src/components/atoms/ProgressBar/index.tsx (hunk 6 above)
Comment (pedrohbl03) [reply src/components/atoms/ProgressBar/index.tsx:21]: How about this way?

**Pull request merged**